src/doc/build-script.md

```diff
@@ -58,15 +60,21 @@ cargo:include=/path/to/foo/include
 There are a few special keys that Cargo recognizes, some affecting how the
 crate is built:
 
-* `rustc-link-lib` indicates that the specified value should be passed to the
-  compiler as a `-l` flag.
-* `rustc-link-search` indicates the specified value should be passed to the
-  compiler as a `-L` flag.
-* `rustc-cfg` indicates that the specified directive will be passed as a `--cfg`
-  flag to the compiler. This is often useful for performing compile-time
+* `rustc-link-lib=[KIND=]NAME` indicates that the specified value is a library
+  name and should be passed to the compiler as a `-l` flag. The optional `KIND`
+  can be one of `dependency`, `crate`, `native`, `framework` or `all` (the
```

Comment:
I don't think `dependency` or `crate` is possible for `-l`, right?

Also, can we refer to "official rustc man pages" or something like that here? I'd hope to avoid duplication of documentation if possible.